src/utils.ts

```diff
@@ -9,8 +10,7 @@ export function arrBufToStr(buf: ArrayBuffer, charSize: CharSize): string {
 }
 
 export function arrBufToBase64(buf: ArrayBuffer): string {
-  const str = arrBufToStr(buf, 8)
-  return Buffer.from(str, 'binary').toString('base64')
+  return base64.fromByteArray(new Uint8Array(buf))
```

Comment:
```suggestion
  return uint8arrays.toString(new Uint8Array(buf), "base64")
```

Can we use the `uint8arrays` package instead? It's already a dependency in our stack generally (it's from the js-ipfs folks), and it's more explicit.
We should make it the somewhat go-to package to work with Uint8Arrays, it includes other in the future potentially really useful functions like `equals`, `fromString` and `concat`, and supports lots of different encodings, e.g. `base64`, `base64pad`, `base64url`.

Reply:
We can definitely use `uint8arrays` yeah, I didn't know it could do base64 as well. I'll switch 👍 
Yeah I ran the tests locally. I'll see if I can if we can the CI in another PR, definitely before publishing this.